play-ebean/src/main/java/play/db/ebean/ModelsConfigLoader.java

```diff
@@ -18,7 +18,8 @@
 
     @Override
     public  Map<String, List<String>> apply(ClassLoader classLoader) {
-        Environment env = new Environment(new File("."), classLoader, Mode.PROD);
+        // Using TEST mode is the only way to load configuration without failing if application.conf doesn't exist
+        Environment env = new Environment(new File("."), classLoader, Mode.TEST);
         Configuration config = Configuration.load(env);
```

Comment:
I tried that first.  It's not accessible from Java.  I don't think the Scala compiler generates static methods for private methods.